src/vignette/storage/local.clj

```diff
@@ -3,6 +3,7 @@
             [clojure.java.shell :as sh]
             [pantomime.mime :refer [mime-type-of]]
             [vignette.media-types :as mt]
+            [digest]
```

Comment:
Minor: We normally either `:refer :all` or `:refer [fn-i-want-to-use other-fn-i-want-to-use]` to make it clear what we are using.